test/unit/math/mix/scal/fun/F32_test.cpp

```diff
@@ -1,4 +1,7 @@
 #include <stan/math/mix/scal.hpp>
+#include <stan/math/mix/mat.hpp>
```

Comment:
1. this file shouldn't include `test/unit/math/rev/mat/fun/util.hpp`. If it does, perhaps we should think about where this file goes and/or what it includes.
2. look at lines 2-5. There are 3 includes that are the same.